docs/index.md

```diff
@@ -11,15 +11,15 @@ Let's say you want to flip a switch in prod. Follow the following procedure:
 3. Edit the `configs/bedrock-prod.env` file to add the name and value you need. You'd add a line like `SWITCH_DO_PHRASING=on` to set that key and value in the environment for prod. Or you can find the line that already has the variable you need and change the value if it already exists.
 4. Commit the change.
 5. Push the change to a branch on your fork.
-6. Submit a pull-request against the `mozmar` repo.
+6. Submit a pull-request against the `www-config` repo.
 7. Ask for a review in the `#www` IRC channel or in the PR itself.
```

Comment:
Hmmm... what I meant was "against the origin repo", or "the mozmar/www-config repo". Not sure the best and most succinct wording to get across "submit the PR against the primary repo".